CONTRIBUTING.md

```diff
@@ -66,6 +66,90 @@ Clone the repository to your local machine from the Azure DevOps endpoint.
 
 - For the real time live syncing to work, our service needs to know your publisher id and your extension's unique key. To enable real time updates for your test extension, please [reach out to us](https://github.com/microsoft/vsts-extension-retrospectives/issues) with your publisher id and the unique key of your extension. [Instructions on how to download the unique key](https://docs.microsoft.com/en-us/azure/devops/extend/develop/auth?view=vsts#get-your-extensions-key).
 
+### Test using Hot Reload and Debug
+
+Test changes by directly load changes locally without having to re-package and re-publish the extension in the marketplace.
+
+**Note:** You will need [Visual Studio Code](https://code.visualstudio.com/download), [Firefox](https://www.mozilla.org/en-US/firefox/) and the [Debugger for Firefox](https://marketplace.visualstudio.com/items?itemName=firefox-devtools.vscode-firefox-debug) VS Code extension.
+
+- In the 'RetrospectiveExtension.Frontend' folder, create the 'vss-extension-dev.json' file using the template.
+
+- Update the 'webpack.config.js' to enable source maps. Set the devtool property to `inline-source-map`. Also set devServer.https to true and devServer.port to 3000.
+
+  ```js
+  module.exports = {
+    devtool: 'inline-source-map',
+    devServer: {
+      https: true,
+      port: 3000,
+      static: {
+        directory: path.join(__dirname),
+      }
+    },
+  ...
+  ```
+
+- Set `output.publicPath` to `/dist/` in the webpack.config.json file. This will allow webpack to serve files from `localhost:3000/dist`.
```

Comment:
```suggestion
- Set `output.publicPath` to `/dist/` in the webpack.config.json file. This will allow webpack to serve files from `https://localhost:3000/dist`.
```